build.ps1

```diff
@@ -137,7 +137,7 @@ $appCommonPackageVersion = "3.0.0"
 Import-Module -Name "$PSScriptRoot/eng/build-helpers.psm1" -Force
 Import-Module -Name "$PSScriptRoot/eng/package-manager.psm1" -Force
 Import-Module -Name "$PSScriptRoot/eng/database-manager.psm1" -Force
-function Clean {
+function DotNetClean {
```

Comment:
I was getting an error with this function before renaming it (`Missing statement block after 'clean'.`). Maybe due to having the latest PowerShell? Seems there is a built-in function called `clean` now, though I cannot figure out its purpose.